.travis.yml

```diff
@@ -4,7 +4,6 @@ node_js:
 - 6
 
 python:
-- 2.7
```

Comment:
💯

Could you please update the classifiers in `setup.py` to reflect that the next release will not be 2.7 compatible?

Reply:
I will try to do this as well as the other request (update the README) either this evening or ASAP tomorrow morning. Sorry for the slow reply on my part; I got distracted by my other project besides this at work.